src/pages/MapPage/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import { Redirect } from 'react-router-dom';
+import { useSnackbar } from 'notistack';
+import { useAppSelector } from 'modules/hooks';
+import useFetch from 'hooks/useFetch';
+import Loading from 'components/shared/Loading/Loading';
+import ROUTE from 'constants/routes';
+import { API_STATUS, END_POINT } from 'constants/api';
+import { ALERT_MESSAGE } from 'constants/messages';
+import { Line, Station, User } from 'types';
+import Styled from './styles';
+
+const MapPage = () => {
+  const user: User | undefined = useAppSelector((state) => state.authSlice.data);
+
+  if (!user) return <Redirect to={ROUTE.HOME} />;
+
+  const [getLinesAsync, getLinesLoading] = useFetch('GET');
+  const [lines, setLines] = useState<Line[]>([]);
+  const [focusedStation, setFocusedStation] = useState<Station>();
+
+  const { enqueueSnackbar } = useSnackbar();
+
+  const getLines = async () => {
+    const res = await getLinesAsync(END_POINT.LINES);
+
+    if (res.status === API_STATUS.REJECTED) {
+      enqueueSnackbar(ALERT_MESSAGE.FAIL_TO_GET_STATIONS);
+    } else if (res.status === API_STATUS.FULFILLED) {
+      setLines(res.data.reverse());
+    }
+  };
+
+  const isTransfer = (station: Station) => station.includedLines.length > 1;
+
+  const checkSameStations = (targetStation: Station) => {
+    if (!isTransfer(targetStation)) return;
+
+    setFocusedStation(targetStation);
+  };
+
+  useEffect(() => {
+    const fetchLines = async () => {
+      await getLines();
+    };
+
+    fetchLines();
```

Comment:
getLines 이름을 fetchLines로 바꾸고 그냥
```suggestion
    fetchLines();
```

만해도 되지않을까요? 👀

Reply:
헉 그렇군요...!!
`getLines`가 async 함수라서 `useEffect` 내부에서도 당연히 `async-await`을 붙여줘야 한다고 생각했습니다. 바보같았군요 😭 
이제라도 알게 돼서 다행입니다... 감사합니닷